app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/ChannelLayoutManager.kt

```diff
@@ -60,6 +60,21 @@ class ChannelLayoutManager(
         return super.onRequestChildFocus(parent, state, child, focused)
     }
 
+    // By default, moving left when the first element is focused or right when the last element
```

Comment:
nit: put comment inside the block.

If someone else needs to override this function for separate functionality, I'm concerned they may just add it in here without moving the comment inside to distinguish your code from theirs.